docs/tutorials/multimodal/document_prediction/pdf_classification.ipynb

```diff
@@ -230,16 +243,9 @@
   }
  ],
  "metadata": {
-  "kernelspec": {
-   "display_name": "agt",
-   "language": "python",
-   "name": "python3"
-  },
   "language_info": {
-   "name": "python",
-   "version": "3.10.14"
-  },
-  "orig_nbformat": 4
+   "name": "python"
+  }
```

Comment:
why did we remove the version specification?

Reply:
Would it work on all the versions?

Reply:
Yes, removing the specific version is to be compatible for all versions.